### PR TITLE
Topic fix  some tests

### PR DESCRIPTION
**Pull request by telephon**
## Purpose and Motivation

NodeProxy tests that failed because of wrong assumptions in the test.

## Types of changes

- Bug fix

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] This PR is ready for review

### CODE REVIEW

File: testsuite/classlibrary/TestNodeProxy.sc [hunk 1]
```diff
@@ -267,7 +267,7 @@ TestNodeProxy : UnitTest {
 		timeout.stop;
 		resp.free;
 
-		this.assertEquals(result, proxy.source, "after the crossfade from a ugen function to a value the bus should have this value");
+		this.assertFloatEquals(result, proxy.source, "after the crossfade from a ugen function to a value the bus should have this value");
```
Comment (dyfer) [inline testsuite/classlibrary/TestNodeProxy.sc:270]: Did `test_control_fade` fail before? If not, what's the rationale for this change?

File: testsuite/classlibrary/TestNodeProxy_Server.sc [hunk 2]
```diff
@@ -71,6 +71,7 @@ TestNodeProxy_Server : UnitTest {
 		// ... and also afterwards, we force an update
 		server.sendStatusMsg;
 		server.sync;
+		0.8.wait;
```
Comment (dyfer) [inline testsuite/classlibrary/TestNodeProxy_Server.sc:74]: Is this really the only way to fix this? This is a rather lengthy wait. Could this be addressed by sending some other message to the server and running another server.sync? Or maybe trying the shorter wait time? 

If you remove the exclusion from `gha_test_run_proto.json` we'll be able to see how the shorter wait time performs in the CI (which is arguably not tuned for audio/low-latency performance).

Comment (telephon) [reply testsuite/classlibrary/TestNodeProxy_Server.sc:74]: I agree. I find it weird that this doesn't work faster, but it has to do with the time the server takes to report the deallocation of the synth def.

Comment (telephon) [reply testsuite/classlibrary/TestNodeProxy_Server.sc]: I've found another solution: just fix the wrong order in the `sync` …

**Review by adcxyz — APPROVED** [review]

**Comment by dyfer** [comment]
Hey @adcxyz 
Maybe I should've mark this as "changes requested", my bad, but if you look at my comments, some of them were not addressed. I'd prefer to wait for outstanding discussion to be resolved before merging a PR, thanks!

**Comment by adcxyz** [comment]
@dyfer 
Sorry, read the discussion too quickly, and thought @telephon had addressed your concerns.
What is still open, enabling these tests again, 
and anything else?

**Comment by dyfer** [comment]
No worries @adcxyz 
Yes, the main point was to remove fixed tests from the "skipped" list in `gha_test_run_proto.json`. Aside from that I asked about the change in `test_control_fade` but that change is minor so I assume it's fine.

**Comment by dyfer** [comment]
`test_synthDef_isReleased_afterFree` got enabled in #5702
